src/main/java/com/iridium/iridiumskyblock/configs/Inventories.java

```diff
@@ -42,6 +44,7 @@ public class Inventories extends com.iridium.iridiumteams.configs.Inventories {
             .put("is boosters", new Item(XMaterial.EXPERIENCE_BOTTLE, 23, 1, "&9&lIsland Boosters", Collections.singletonList("&7View your island boosters")))
             .put("is home", new Item(XMaterial.WHITE_BED, 13, 1, "&9&lIsland Home", Collections.singletonList("&7Teleport to your island home")))
             .put("is members", new Item(XMaterial.PLAYER_HEAD, 14, 1, "&9&lIsland Members", "Peaches_MLG", Collections.singletonList("&7View your island members")))
+            .put("is biomes", new Item(XMaterial.GRASS_BLOCK, 15, 1, "&9&lIsland Biomes", Collections.singletonList("&7Change your island biome")))
```

Comment:
Unsure how this would make the /is menu look? currently it is symetrical, adding this might make it look weird no?